packages/cms-editor/i18next-parser.config.mjs

```diff
@@ -2,7 +2,7 @@ export default {
   defaultNamespace: 'cms-editor',
   defaultValue: '__MISSING_TRANSLATION__',
   keepRemoved: false,
-  locales: ['en', 'de'],
+  locales: ['en', 'de', 'jp'],
```

Comment:
why do you need to add it here?
I think it is enough to add it the the init of the standalone and do not have it here where it generates the translation files.